client/components/text-area-input.jsx

```diff
@@ -36,9 +43,12 @@ export default class TextAreaInput extends React.Component {
             id="message-input"
             style={textAreaStyle}
             placeholder="Send a message. . . "
+            value={this.props.messageValue}
             onChange={this.handleChange}>
           </textarea>
-          <i className="fas fa-chevron-circle-right send-message-icon"></i>
+          <i
+          onClick={this.sendMessage}
+          className="fas fa-chevron-circle-right send-message-icon"></i>
```

Comment:
You should be using a `<form>` and a submit event listener not a click listener. Use a `button` for submitting. It can be styled to look like an icon. Just don't listen for clicks when you mean form submission.